### PR TITLE
Bootstrap package

### PR DESCRIPTION
**Pull request by potemkin-git**
https://github.com/rgsystemes/kb/issues/7118

### CODE REVIEW

File: tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  Write-Output "choco install rgsupervision --params "'/Token:value(string) /Node:value(int)'""
+  Write-Output "If you do not have your deployment token yet, check your Deployment section within your user profile management page."
+  Write-Output "You will then be able to finish the installation by running the following command:"
+  Write-Output "$($exeFilePath) --action register --login token@token.tk --password <YourToken> --node #$($pp['Node'])"
+} else {
+  Write-Output "Starting agent install using deployment token: $($pp['Token'])"
+
+  if ($pp['Staging'] -eq "1") {
+    Write-Output "Deploying (Staging) on node #$($pp['Node'])"
+    & $exeFilePath --action register --login "token@token.tk" --password "$($pp['Token'])" --node "#$($pp['Node'])" --expected-host-name lisa.staging.rg.gg
```
Comment (sandvige) [inline tools/chocolateyinstall.ps1]: nop!

File: README.md [hunk 2]
```diff
@@ -1 +1,28 @@
...
+
+A user account will be required prior deploying, in order to authenticate with your **deployment token**. 
+This token can be generated from the user profile page within dashboard (parameters tab).
+
+Don't have any account ? Visit https://www.rgsystem.com for more information.
+
+## Package parameters
+The following package parameters must be set:
+* `/Token:` (string) User deployment token used for authentication while registering agent (required)
+* `/Node:`  (integer) Node ID where agent will be located within dashboard (default is user root node)
```
Comment (sandvige) [inline README.md]: Default means optional?

File: tools/chocolateyinstall.ps1 (hunk 1 above)
Comment (sandvige) [reply tools/chocolateyinstall.ps1]: I'd prefer you use $pp['ExpectedHostName'] instead

**Review by sandvige — CHANGES_REQUESTED** [review]
Excellent ;)

File: tools/chocolateyinstall.ps1 (hunk 1 above)
Comment (potemkin-git) [reply tools/chocolateyinstall.ps1]: for whole "Staging" option ?

Comment (sandvige) [reply tools/chocolateyinstall.ps1]:
``` ps
if ($pp['ExpectedHostName'] -eq "1") {
    Write-Output "Deploying (custom hostname) on node #$($pp['Node'])"
        & $exeFilePath --action register --login "token@token.tk" --password "$($pp['Token'])" --node #$($pp['Node'])" --expected-host-name "$($pp['ExpectedHostName'])"
} else {
    Write-Output "Deploying on node #$($pp['Node'])"
        & $exeFilePath --action register --login "token@token.tk" --password "$($pp['Token'])" --node "#$($pp['Node'])"
}
```

File: README.md (hunk 2 above)
Comment (potemkin-git) [reply README.md]: Yup, changing wording, seems unclear 😇

**Review by sandvige — APPROVED** [review]

**Pull request merged**